packages/react-storage/src/components/StorageBrowser/__tests__/Controller.spec.tsx

```diff
@@ -57,18 +115,18 @@ describe('Controller', () => {
       .mockReturnValueOnce(loadingState)
       .mockReturnValue(resolvedState);
 
-    const { rerender } = render(<Controller />);
+    const { rerender } = render(<Provider />);
```

Comment:
I think since `Controller` is already inside the `Provider` after calling `createProvider`, it makes sense to just render the `Provider`. Let me know if I should do something else though